src/index.js

```diff
@@ -42,20 +50,61 @@ Promise.all([recievedUsersData, recievedRoomsData, recievedBookingsData])
 
 
 function startApp() {
-
+  showUsers();
 }
 
 function validateLogin(event) {
   event.preventDefault();
-  if(usernameInput.value === 'manager' && userPasswordInput.value === 'overlook2020') {
+  if (usernameInput.value === 'manager' && userPasswordInput.value === 'overlook2020') {
     currentUser = new Manager();
-    // helper function to show customer dashboard
+    openManagerDash();
+    console.log(currentUser);
+    // helper funtion to show manager dashboard
   } else if (usernameInput.value.slice(0, 8) === 'customer' && userPasswordInput.value === 'overlook2020') {
     let currentId = Number(usernameInput.value.slice(8));
     let userById = userData.find(user => user.id === currentId);
     currentUser = new User(userById)
-    // helper funtion to show manager dashboard
-  }else {
+    // helper function to show customer dashboard
+  } else {
     alert("Appologies, the username or password you have entered does not match any we have on file. Please try again!")
   }
 }
+
+function openManagerDash() {
+  managerDash.classList.remove('hidden');
+  nav.classList.remove('hidden');
+  modal.classList.add('hidden');
+  modalOverlay.classList.add('hidden');
+  numberOfRoomsAvailable(bookingData, roomData, "2020/04/22");
```

Comment:
The date "2020/04/22" was purely for testing purposes. Need to write a function to get today's date and throw that in instead.